sources/packages/backend/apps/test-db-seeding/src/clean-db/clean-db.ts

```diff
@@ -8,5 +8,6 @@ export class CleanDatabase {
   async cleanDatabase(): Promise<void> {
     // Drops the database and all its data.It will erase all your database tables and their data.
     await this.dataSource.dropDatabase();
+    await this.dataSource.query("DROP EXTENSION pg_trgm");
```

Comment:
Do we need to have something like this below ? to avoid errors if the extension is not exists?
`await dataSource.query("DROP EXTENSION IF EXISTS pg_trgm");`

Reply:
I get your point @guru-aot . But we devs recently decided to stop using the `IF EXISTS` and `ID NOT EXISTS` to be aware of the side effects and handle them. That was the reason why I avoided.

Reply:
I get that @dheepak-aot, I am not going to stress on this but for this exceptional scenario, I was validating if we need it or not

Reply:
There was a decision to stop using the `IF EXISTS` for migrations because it is a controlled script execution where the up/down must be precise and if it is changing something the changed item (column/index/whatever) should be there.
For this situation, the same may not be applied, for instance, if executing the `clean database` twice will make the second execution fail.